corehq/pillows/mappings/app_mapping.py

```diff
@@ -49,6 +49,7 @@
                 'copy_history': {'type': 'string'},
                 'copy_of': {'type': 'string'},
                 'cp_is_active': {'type': 'boolean'},
+                'created_from_template': {'type': 'string'},
```

Comment:
@czue or @snopoke Do you know if we already have some better way of doing this? Presumably it could compute that from the hash of the mapping contents the way the pillowtop checkpoint doc does. Ideally it'd be the same hash for easier cross referencing